dispatcher/backend/src/routes/users/user.py

```diff
@@ -6,6 +6,7 @@
 from marshmallow import ValidationError
 
 from common.mongo import Users
+from common.mongo import Workers
```

Comment:
You should share that import with the previous one

Reply:
@rgaudin I have done the specified changes